### DISCUSSION TITLE
Mas posing?? (idk how to say it)

### DISCUSSION THREAD

**Discussion by mizzimizzu** [Monika-After-Story/MonikaModDev#10192] [QA - code and content creation (technical)]
So you know how when you add dialogue, before the sentence you want monika to say, in the code you use a jumble of letters and numbers to make the pose you want monika to be in? if you dont understand, i put a picture example. 
 is there certain combinations that don't work? I tried to make dialogue and i thought that you could do any combination...buttt maybe im wrong?
![Screenshot 2024-02-02 171749](https://github.com/Monika-After-Story/MonikaModDev/assets/155788720/8b82d007-3015-4ca1-b609-14e77ca92f9b)

**Comment by Wingdinggaster656** [comment]
These codes have some certain rules, not just simple combinations of letters and numbers. You may want to check out this file: https://github.com/Monika-After-Story/MonikaModDev/blob/master/Monika%20After%20Story/game/dev/dev_exp_previewer.rpy
